raw_linux.go

```diff
@@ -26,7 +26,10 @@ type packetConn struct {
 	s   socket
 	pbe uint16
 
-	// Timeouts set via Set{Read,}Deadline, guarded by mutex
+	// Should timeouts be set at all?
```

Comment:
Maybe specifying that it's the Receive timeout only ?

Reply:
This package technically has no write timeouts at the moment anyway, so YOLO!